src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md

```diff
@@ -0,0 +1,79 @@
+---
+path: "/docs/running-simbank-tests-cli"
+title: "Running the SimBank tests using the CLI"
+---
+
+Galasa SimBank comes with a selection of prepared Galasa tests:
+
+- A basic Installation Verification Test (IVT) which logs on to SimBank  - `SimBankIVT.java`.
+- A test that updates an account using web services and examines the changes with 3270 screens - `BasicAccountCreditTest.java`.
+- A test that uses a provisioned account object to perform the same test as `BasicAccountCreditTest.java` in an improved test design - `ProvisionedAccountCreditTests.java`.
+- A test that exercises the z/OS Batch Manager by simulating the submission of a JCL job to add a number of accounts to the SimBank system - `BatchAccountsOpenTest.java`.
+
+The following section explains how to run the `SimBankIVT` test class by using the CLI. 
+
+## Running the SimBank IVT test class by using the CLI
+
+The SimBank tests are held in the <a href="https://github.com/galasa-dev/simplatform" target="_blank"> Galasa simplatform repository</a> in GitHub. To start running the tests you need to clone the repository and launch SimBank so that the tests are available on your local machine. To find out how to clone the cli repository and launch the SimBank application, follow the instructions in the `Launching SimBank` section in the [Exploring Galasa SimBank using the CLI]() documentation.
+
+After cloning the repository and launching SimBank, complete the following steps to run the SimBankIVT test that is provided with Galasa. The following example uses SimBank OBR version `0.25.0` and Galasa OBR version `0.29.0`.
+
+1. In a new terminal, navigate to your `simplatform` directory and run the following command to install and build your project:
+
+```
+mvn install
+```
+
+2. In the same `simplatform` directory, run the following command to run the SimBankIVT test:
+
+On Mac or Unix:
+
+```
+galasactl runs submit local --log - \
+--obr mvn:dev.galasa/dev.galasa.simbank.obr/0.25.0/obr \
+--obr mvn:dev.galasa/dev.galasa.uber.obr/0.29.0/obr \
+--class dev.galasa.simbank.tests/dev.galasa.simbank.tests.SimBankIVT 
```

Comment:
I couldn't get this to work. I tried a few different directories to see if that helped but no luck. It might be useful to include a reminder to run `galasactl local init` (really weirdly the command fails to run if the `bootstrap.properties` file is missing but then tries to create it when it does exist!)